lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -79,43 +78,43 @@ def _handle_key(key):
     return key
 
 
-class TimerTornado(backend_bases.TimerBase):
```

Comment:
This can't be removed without a deprecation; you probably want to move the import locally at the least.